CHANGES.RST

```diff
@@ -1,3 +1,8 @@
+3.1.0 (2018-06-04)
+=================
+
+- Adds support for session decisions to [Decisions API](https://siftscience.com/developers/docs/curl/decisions-api)
```

Comment:
Is it possible to have the same wording between all clients ? Maybe "Add support for get latest session decisions"?